### ISSUE TITLE
Accordion BS3 Gives error in DNN 9.6.0

### ISSUE THREAD

**Issue by TopCat6** [sachatrauwaen/OpenContent#117]
The Bootstrap3 Accordion template gives an error when you click on the item to expand it. Expansion fails. Console error is:

```
TypeError: can't convert o to string                                       bootstrap.min.js:6:8739
    test self-hosted:2737
    l http://localhost/dotnetnuke/Portals/_default/skins/xcillion/bootstrap/js/bootstrap.min.js?cdv=1344:6
    jQuery 2
        each
        each
    l http://localhost/dotnetnuke/Portals/_default/skins/xcillion/bootstrap/js/bootstrap.min.js?cdv=1344:6
    <anonymous> http://localhost/dotnetnuke/Portals/_default/skins/xcillion/bootstrap/js/bootstrap.min.js?cdv=1344:6
    jQuery 2
        dispatch
        handle
```
Tried on 3 different sites with 2 different skins.
OC 4.4, DNN 9.6

**Comment by TopCat6** [comment]
Found out there is a bug in jQuery 3.5.0 library which they patched in 3.5.1.  As a workaround I successfully replaced the files in \Resources\Libraries\jQuery\03_05_00\ with the new files from 3.5.1 (after renaming appropriately the jquery and map files), then restarted server and cleared cache.

It all works fine now.

**Comment by Timo-Breumelhof** [comment]
ok thanks for the feedback :-)
